robbot.py

```diff
@@ -26,6 +16,33 @@
 
 bot = commands.Bot(command_prefix='.')
 
+'''
+    Cog Functions & Commands
+'''
+def load_cogs():
+    for files in os.listdir('./cogs'):
+        if files.endswith('.py'):              
```

Comment:
Since `files` refers to a single element in the list of filenames returned by `os.listdir()`, not a list itself, the name `file` would be more precise.

```suggestion
    for file in os.listdir('./cogs'):
        if file.endswith('.py'):              
```